runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/internal/catalog/TestCatalogContext.java

```diff
@@ -31,7 +32,7 @@ public TestCatalogContext(
     public CatalogHandler attach(
         CatalogConfig catalog)
     {
-        return new TestCatalogHandler(catalog);
+        return new TestCatalogHandler(TestCatalogConfig.class.cast(catalog.options));
```

Comment:
Please rename `TestCatalogConfig` to `TestCatalogOptionsConfig`.
Feel free to introduce a `TestCatalogConfig` similar to `TestGuardConfig` if needed.